### DISCUSSION TITLE
iPad run out of memory when cloning

### DISCUSSION THREAD

**Discussion by ZJsheep** [Vinzent03/obsidian-git#329] [Problems]
The problem is that when I tried to clone a repo with about 2k files of about 300MB size on iPad it told me that it runs out of memory. The repo is [here](https://github.com/ZJsheep/_Course/tree/main).

I tried a workaround. I downloaded the repo maually into Obsidian and tried making it git, but it told me that file 'config' cannot be opened.

![IMG_0664](https://user-images.githubusercontent.com/73184640/193802399-a7ee0dc2-a9ba-4bea-8cfc-672b2e472859.PNG)

So, will this workaround of doing with existing files take less memory? If it should work, how to solve the problem of 'config' cannot be opened?

**Comment by Vinzent03** [comment]
It should work, copying an existing repo with the .git folder to an iPad. So that's strange. I think you didn't copy everything.

**Reply by Gabaa** [reply]
I can confirm! I got around the cloning issue on iPad by transferring the entire folder from my iPhone to my iPad. Subsequent pulls seem to work fine! (Obviously, you need to have the full folder stored somewhere - perhaps you can upload it to iCloud from your PC?)
